capella2polarion/data_model/work_item_attachments.py

```diff
@@ -143,9 +151,20 @@ def content_checksum(self) -> str:
                     styleclass_map, sort_keys=True, separators=(",", ":")
                 )
 
-                self._checksum = hashlib.sha256(
-                    f"{input_str};{styleclass_str}".encode()
-                ).hexdigest()
+                content = self.content_bytes
```

Comment:
This will cost us performance because every image always needs to be rendered even if the content did not change.

Reply:
Also, do we even need the checksum at all if rendering failed? I think we could just as well make it a static not-a-real-checksum string. The failure path could become a bit trickier, as we'd have to take care not to repeatedly overwrite errors with errors. (Since we'll always attempt to render after a previous failure, we'll presumably also always try to upload, unless we add another check in the failed-to-render path.)